### ISSUE TITLE
Netapp Trident not parsing response from NetApp OnTAP 9.3 after PV creation

### ISSUE THREAD

**Issue by CalvinHartwell** [NetApp/trident#125]
Hi all,

Not a critical bug but Trident is not parsing some of the responses back from the NetApp OnTAP device correctly after PV creation: 

```
time="2018-05-16T11:47:30Z" level=warning msg="Error unmarshaling response body. xml: name \"volume-key\" in tag of azgo.VolumeModifyIterInfoType.VolumeKeyPtr conflicts with name \"volume-attributes\" in *azgo.VolumeAttributesType.XMLName" body="<?xml version='1.0' encoding='UTF-8' ?>\n<!DOCTYPE netapp SYSTEM 'file:/etc/netapp_gx.dtd'>\n<netapp version='1.130' xmlns='http://www.netapp.com/filer/admin'>\n<results status=\"passed\"><success-list><volume-modify-iter-info><volume-key><volume-attributes><volume-id-attributes><name>cats_kittens_lions_tigers_e100c</name><owning-vserver-name>unckc</owning-vserver-name></volume-id-attributes></volume-attributes></volume-key></volume-modify-iter-info></success-list><failure-list></failure-list><num-failed>0</num-failed><num-succeeded>1</num-succeeded></results></netapp>"
```

RAW: [https://gist.githubusercontent.com/CalvinHartwell/e46c5c8d31a3670546c160c5da1abb39/raw/f1f599d2ec6993f87e020a6f318c12627a4843ec/gistfile1.txt](https://gist.githubusercontent.com/CalvinHartwell/e46c5c8d31a3670546c160c5da1abb39/raw/f1f599d2ec6993f87e020a6f318c12627a4843ec/gistfile1.txt)

Any ideas? 

Cheers,

- Calvin

**Comment by adkerr** [comment]
@CalvinHartwell this is a known issue and the message is benign. It’s a side effect of some autogenerated api code.

**Comment by clintonk** [comment]
@CalvinHartwell Which version of Trident are you using?  This log warning should have been suppressed in 18.04.

**Comment by korenaren** [comment]
As @clintonk mentioned, this is fixed in the 18.04 release. Closing this issue.

**Comment by CalvinHartwell** [comment]
Thanks for fixing this!
